projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -76,28 +76,26 @@ export class NavigationListComponent implements OnChanges {
   @Output()
   public readonly navItemSelected: EventEmitter<NavItemLinkConfig> = new EventEmitter();
```

Comment:
nit: given the similarity between the outputs (which I do think is worthy of two separate outputs), a rename here to navItemClick might be in order - basically one that triggers on explicit click, the other on any change, no matter the cause.